lsl/imaging/utils.py

```diff
@@ -1544,21 +1544,35 @@ def bm_image(self, center=(0,0), term=None, weighting='natural', local_fraction=
             return [self._gen_img(b, center=center, weighting=weighting, local_fraction=local_fraction, robust=robust, taper=taper) for b in self.bm]
 
 
-def build_gridded_image(data_set, size=80, res=0.50, wres=0.10, pol='XX', chan=None, im=None, verbose=True):
+def build_gridded_image(data_set, uv_size=80, uv_res=0.50, im_size=None, im_res=None, wres=0.10, pol='XX',chan=None, im=None, verbose=True):
```

Comment:
`size` should probably remain `size` to prevent breaking things outside of LSL.  Unless we're going for a new major release.